vendor/modules.txt

```diff
@@ -205,6 +205,9 @@ github.com/cespare/xxhash
 # github.com/cespare/xxhash/v2 v2.1.2
 ## explicit; go 1.11
 github.com/cespare/xxhash/v2
+# github.com/cloudflare/cloudflare-go v0.27.0 => github.com/cyriltovena/cloudflare-go v0.27.1-0.20211118103540-ff77400bcb93
```

Comment:
Do you plan on upstreaming these changes?

Reply:
Yes https://github.com/cloudflare/cloudflare-go/pull/743